### PR TITLE
SLI-427 Show current rule param value in rule description tab

### PR DESCRIPTION
**Pull request by kirill-knize-sonarsource**

### CODE REVIEW

**Comment by damien-urruty-sonarsource** [comment]
@kirill-knize-sonarsource I added a link to open rule settings from the description, could you have another look please ?

**Comment by damien-urruty-sonarsource** [comment]
@jblievremont maybe you can also have a look, we relied on your work on VSCode

File: src/main/java/org/sonarlint/intellij/ui/SonarLintRulePanel.java [hunk 1]
```diff
@@ -226,6 +263,53 @@ private static String fixPreformatedText(String htmlBody) {
     return builder.toString();
   }
 
+  private String renderRuleParams(StandaloneRuleDetails ruleDetails) {
+    if (!ruleDetails.paramDetails().isEmpty()) {
+      StyleSheet styleSheet = kit.getStyleSheet();
+
+      styleSheet.addRule(".rule-params { border: none; border-collapse: collapse; padding: 1em; }");
+      styleSheet.addRule(".rule-params caption { font-size: 16px; font-weight: 400; text-align: left; margin-bottom: 16px}");
+      styleSheet.addRule(".rule-params thead td { vertical-align: top; padding-left: 0; padding-bottom: 1em; font-style: italic }");
```
Comment (jblievremont) [inline src/main/java/org/sonarlint/intellij/ui/SonarLintRulePanel.java:272]: It looks like some of these styles are not applied, maybe we can do something more visually appealing within the constraints of the Swing HTML renderer.

image

**Comment by jblievremont** [comment]
I tried to address most rendering issues, too bad we can't use JCEF yet :sweat_smile: 

----

Screenshot from 2020-08-07 16-04-34

----

**Pull request merged**